### PR TITLE
[cases] add rvv_bench

### PR DESCRIPTION
**Pull request by SharzyL**

### CODE REVIEW

**Comment by sequencer** [comment]
Thanks @SharzyL for adding @camel-cdr tests here!
These benchs will be the guide for T1 performance tuning.
T1 will use it for CI in the future, after T1 successfully passed all workloads provided by @camel-cdr, we will add that repo as dependency.

**Review by sequencer — DISMISSED** [review]
Please add corresponding tests to CI

File: .github/cases/blastoise/default.json [hunk 1]
```diff
@@ -498,5 +498,16 @@
   "codegen.vfredosum_vs": 142145,
   "codegen.vfredusum_vs": 99859,
   "codegen.vfredmax_vs": 99859,
-  "codegen.vfredmin_vs": 99859
+  "codegen.vfredmin_vs": 99859,
+  "rvv_bench.ascii_to_utf16": 1541742,
+  "rvv_bench.ascii_to_utf32": 673956,
+  "rvv_bench.byteswap": 3456435,
+  "rvv_bench.chacha20": 0,
```
Comment (sequencer) [inline .github/cases/blastoise/default.json:505]: cc @camel-cdr, it seems chacha20 and poly1305 doesn't go to auto vectorization?

Comment (camel-cdr) [reply .github/cases/blastoise/default.json:505]: I didn't explicitly disable it, but it doesn't get autovectorized by todays compilers.

**Comment by sequencer** [comment]
why CI doesn't work...

**Comment by sequencer** [comment]
why CI doesn't work...

**Comment by sequencer** [comment]
@SharzyL please reopen it, github failed to recover:(

**Pull request closed without merging**